invenio_cli/commands/packages.py

```diff
@@ -37,6 +37,44 @@ def install_packages(packages):
 
         return steps
 
+    @staticmethod
+    def outdated_packages():
+        """Steps to show outdated packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + ['--outdated']
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Installing python dependencies..."
```

Comment:
moderate: Is this message correct? Is not installing anything, right? It's calculating outdated dependencies...